### PR TITLE
Reload

### PR DESCRIPTION
**Pull request by Raisa-12**
### Functionality:
when we reload the tasks page for deleted tasks, it keeps reloading
fix it

### Solution:
changes done in the showDeleted section of sprints-details.component.ts

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
go to tasks page for deleted tasks,click on it,it is getting reloaded and is showing the tasks u deleted

### CODE REVIEW

**Review by Pendyala-thanvi — DISMISSED** [review]

**Review by viv9k — APPROVED** [review]
Approved

File: src/app/body/board/sprint-details/sprint-details.component.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ export class SprintDetailsComponent implements OnInit {
   }
 
   showDeleted() {
```
Comment (sanjaykrishna1203) [inline src/app/body/board/sprint-details/sprint-details.component.ts:96]: Won't this change show the tasks of the active sprint instead of Deleted Tasks

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

File: src/app/body/board/sprint-details/sprint-details.component.ts [hunk 2]
```diff
@@ -94,7 +94,7 @@ export class SprintDetailsComponent implements OnInit {
...
-    this.currentSprint.emit(-2);
+    this.currentSprint.emit(this.currentSprintNumber);
```
Comment (sanjaykrishna1203) [inline src/app/body/board/sprint-details/sprint-details.component.ts:97]: This will show the tasks of the Active Sprint instead of Deleted Tasks

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

**Review by viv9k — CHANGES_REQUESTED** [review]
Not a good solution, It won't work, Rather it will create more issues.

File: src/app/body/board/sprint-details/sprint-details.component.ts (hunk 2 above)
Comment (viv9k) [reply src/app/body/board/sprint-details/sprint-details.component.ts:97]: Good point @sanjaykrishna1203

**Comment by Raisa-12** [comment]
But I have created a dummy task and deleted it,it is showing in the deleted
task section.

**Pull request closed without merging**